packages/documentation/copy/en/handbook-v2/Object Types.md

```diff
@@ -426,7 +426,6 @@ Since assigning `squareOptions` won't undergo excess property checks, the compil
 interface SquareConfig {
   color?: string;
   width?: number;
-  [propName: string]: any;
```

Comment:
An explanation of this change ---

This is a separate thing. I think this string indexer was a copy-paste mistake that shouldn't have been added here.

This section follows the suggestion to add a string index signature. It starts off saying "One final way to get around these checks, which might be a bit surprising, is to assign the object to another variable...". So _this solution_ does not solve the problem by using a string index signature. However, the index signature is there in the hidden code.

If I'm mistaken about this, please feel free to edit this PR.